src/unity/python/turicreate/visualization/_plot.py

```diff
@@ -29,6 +29,13 @@ def _get_client_app_path():
     if _sys.platform == 'linux2' or _sys.platform == 'linux':
         return _os.path.join(tcviz_dir, 'Turi Create Visualization', 'visualization_client')
 
+def _focus_client_app():
+    scpt = '''
+            delay .5
+            tell application \"Turi Create Visualization\" to activate
+            '''
+    focus = __subprocess._Popen(['osascript', '-'], stdout=_PIPE, stdin=_PIPE)
```

Comment:
I think we have the equivalent of this elsewhere (probably duplicated in the `show` and `explore` methods). It would be nice to refactor to share the implementation -- but not necessary for this PR.